### PR TITLE
updates project to use fractal-fork

### PR DESCRIPTION
**Pull request by joyheron**
This updates the project to use fractal-fork instead of the official fractal image, because the official fractal image has had a major vulnerability for >1 year. It isn't a big change, so it should be possible to migrate back to the official fractal version if a fix is released in the future.

The entire list of changes that differ from the offical image can be read here:
https://github.com/joyheron/fractal#migration-guide-from-frctlfractal-to-fractal-fork

TL;DR different import structure, much less powerful/colorful CLI, no automatic port
 discovery, fewer dependencies, no vulnerabilities

### CODE REVIEW

**Comment by joyheron** [comment]
Note: I generated the `yarn.lock` using `npm install` not `yarn install`.

**Comment by joyheron** [comment]
I've just noticed that the `npm run site` command isn't working. I'm looking into it now.

**Comment by joyheron** [comment]
`npm run site` wasn't working because I overlooked the `fractal -> fractal-fork` replacement. Should be working now.

**Comment by joyheron** [comment]
I've also regenerated the `yarn.lock` file so that it is generated by yarn and not npm

**Review by nanoparsec — APPROVED** [review]
